fdbkubernetesmonitor/certloader/certloader.go

```diff
@@ -0,0 +1,56 @@
+package certloader
```

Comment:
Can we move this into an `internal` package? I don't think we want to expose the cert loader to other users.

Reply:
no problem